Can't compile when including transaction.hpp

Using the test contract below, I'm unable to compile when including `<eosio/transaction.hpp>`.

```
/usr/opt/eosio.cdt/1.6.1/bin/wasm-ld: error: duplicate symbol: eosio::send_deferred(unsigned __int128 const&, eosio::name, char const*, unsigned int, bool)
>>> defined in /tmp/test.cpp.o
>>> defined in /tmp/test.cpp.o
```

```
#include <eosio/contract.hpp>
#include <eosio/transaction.hpp>

using namespace eosio;

namespace test {
    class [[eosio::contract]] test : public contract {
        public:
            test(name receiver, name code, datastream<const char*> ds) : contract(receiver, code, ds) { }

            ACTION testact( name user ) {
                require_auth( user );
            }
    };
}
```

I am blocked on this issue. I am getting the below errors. I am using cmake and I don't think EOS Studio is a workaround in my case.  You can replicate the issue by cloning the following repo and running make (https://github.com/gyftie/gyftie-contracts2).  I am looking for a workaround.  thanks 

/usr/local/Cellar/eosio.cdt/1.6.1/opt/eosio.cdt/bin/wasm-ld: error: duplicate symbol: eosio::send_deferred(unsigned __int128 const&, eosio::name, char const*, unsigned int, bool)
>>> defined in CMakeFiles/gyftietoken.dir/gyftietoken.cpp.obj
>>> defined in CMakeFiles/gyftietoken.dir/__/classes/profile.cpp.obj

/usr/local/Cellar/eosio.cdt/1.6.1/opt/eosio.cdt/bin/wasm-ld: error: duplicate symbol: eosio::send_deferred(unsigned __int128 const&, eosio::name, char const*, unsigned int, bool)
>>> defined in CMakeFiles/gyftietoken.dir/__/classes/profile.cpp.obj
>>> defined in CMakeFiles/gyftietoken.dir/__/classes/gyft.cpp.obj

/usr/local/Cellar/eosio.cdt/1.6.1/opt/eosio.cdt/bin/wasm-ld: error: duplicate symbol: eosio::send_deferred(unsigned __int128 const&, eosio::name, char const*, unsigned int, bool)
>>> defined in CMakeFiles/gyftietoken.dir/__/classes/gyft.cpp.obj
>>> defined in CMakeFiles/gyftietoken.dir/__/classes/proposal.cpp.obj

/usr/local/Cellar/eosio.cdt/1.6.1/opt/eosio.cdt/bin/wasm-ld: error: duplicate symbol: eosio::send_deferred(unsigned __int128 const&, eosio::name, char const*, unsigned int, bool)
>>> defined in CMakeFiles/gyftietoken.dir/__/classes/proposal.cpp.obj
>>> defined in CMakeFiles/gyftietoken.dir/__/classes/reputation.cpp.obj

/usr/local/Cellar/eosio.cdt/1.6.1/opt/eosio.cdt/bin/wasm-ld: error: duplicate symbol: eosio::send_deferred(unsigned __int128 const&, eosio::name, char const*, unsigned int, bool)
>>> defined in CMakeFiles/gyftietoken.dir/__/classes/reputation.cpp.obj
>>> defined in CMakeFiles/gyftietoken.dir/__/classes/lock.cpp.obj

/usr/local/Cellar/eosio.cdt/1.6.1/opt/eosio.cdt/bin/wasm-ld: error: duplicate symbol: eosio::send_deferred(unsigned __int128 const&, eosio::name, char const*, unsigned int, bool)
>>> defined in CMakeFiles/gyftietoken.dir/__/classes/lock.cpp.obj
>>> defined in CMakeFiles/gyftietoken.dir/__/classes/gyftie.cpp.obj
make[5]: *** [gyftietoken/gyftietoken.wasm] Error 255

--- a/libraries/eosiolib/contracts/eosio/transaction.hpp
+++ b/libraries/eosiolib/contracts/eosio/transaction.hpp
@@ -172,7 +172,7 @@ namespace eosio {
     *  @param size - The size of the packed transaction, required for persistence.
     *  @param replace - If true, will replace an existing transaction.
     */
-   void send_deferred(const uint128_t& sender_id, name payer, const char* serialized_transaction, size_t size, bool replace = false) {
+   inline void send_deferred(const uint128_t& sender_id, name payer, const char* serialized_transaction, size_t size, bool replace = false) {
      internal_use_do_not_use::send_deferred(sender_id, payer.value, serialized_transaction, size, replace);
    }
    /**

